pkg/operator/controller/callbacks.go

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
I assume all this will be replaced by #911?

Reply:
yes